wiki/Brand_identity_guidelines/en.md

```diff
@@ -2,39 +2,69 @@
 
 The brand identity guidelines is a set of standards for branding osu!-related projects. All use does not require approval.
 
-[Download the design kit here.](https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view?usp=sharing) The linked design kit contains some outdated assets. Also note that the included PDF is outdated and this page should be used instead.
+## Assets
+
+*Note: Some parts of the design are still work in progress, unless stated otherwise.*\
+*See also: [osu-resources — Assets used by osu!](https://github.com/ppy/osu-resources)*
+
+The [osu! Figma library](https://www.figma.com/file/6m10GiGEncVFWmgOoSyakH/osu!-Figma-Library) contains design assets such as the osu! logo, mod icons, colour palettes, and many more. To obtain them, log into Figma, mark objects for download using rectangular selection, and click the `Export` tab in the top right corner.
+
+The osu! website design is licensed under AGPL version 3 or later. Please see [the licence file](https://github.com/ppy/osu-web/blob/master/LICENCE) for more information.
+
+## osu!
+
+The name of the game, osu!, must not be capitalised. The osu! official branding must not use any spaces:
+
+- Game modes: `osu!`, `osu!catch`, `osu!taiko`, `osu!mania`
+- Projects: `osu!academy`, `osu!talk`
+- Services and applications: `osu!direct`, `osu!store`, `osu!stream`, `osu!tourney`
+- Products and goods: `osu!keyboard`, `osu!supporter`, `osu!tablet`
+
+For all other terms, osu! must be treated as a [qualifying noun](https://en.wikipedia.org/wiki/Noun_adjunct), which means adding a space between osu! and the noun it modifies. Examples:
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
 
 ## osu! cookie logo
 
 ### Cookie usage restrictions
 
 #### Single colour
 
-![](img/usage-single-colour.png)
+::: Infobox
+![](img/usage-single-colour.png "Single colour osu! logo")
+:::
 
 This is the single colour version of the osu! cookie logo. This version of the logo is very versatile and can be adapted to many design styles.
 
 Please use the original file as the base, and do not create the logo from scratch.
 
 #### Full colour
 
-![](img/usage-full-colour.png)
+::: Infobox
+![](img/usage-full-colour.png "Full colour osu! logo")
+:::
 
 This is the full colour version of the osu! cookie logo. The subtle drop shadow is part of the full colour logo.
 
 Please use the logo as provided by the design kit without any modification.
 
 ### Clear space area
 
-![](img/clear-space-area.png)
+::: Infobox
```

Comment:
the paragraph below is so short that the infobox will look kinda wonky

Reply:
I suspect that but hope it will not, otherwise I need to remove the two infoboxes above as well to not make this specific image look out of the place

Reply:
removed